backend/src/main/java/io/kongkham/kcurr/CurrController.java

```diff
@@ -1,23 +1,78 @@
 package io.kongkham.kcurr;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.reactive.function.client.WebClient;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 @RestController
 public class CurrController {
```

Comment:
Perhaps we want to add "/curr" to create namespace for the urls in this controller

Reply:
@RequestMapping